package-lock.json

```diff
@@ -1,7 +1,7 @@
 {
   "name": "@eclipse-sirius/sirius-components-parent",
   "version": "2023.10.9",
-  "lockfileVersion": 2,
+  "lockfileVersion": 3,
```

Comment:
This makes the package-lock incompatible with node 18.7 / npm 8.15 which are the versions currently used.

Reply:
I put back to lockfileVersion version 2

Reply:
Just declaring "this is version 2-compliant" does not make it so. The actual content of the file is very different, and the npm version we currently use (8.15.0) does not interpret this file correctly.